src/field/field.cxx

```diff
@@ -34,21 +34,42 @@
 #include <utils.hxx>
 #include <bout/mesh.hxx>
 
-Field::Field() : fieldmesh(nullptr){
+Field::Field() : fieldmesh(nullptr), fieldCoordinates(nullptr) {
 #if CHECK > 0
   bndry_xin = bndry_xout = bndry_yup = bndry_ydown = true;
 #endif
 }
 
-Field::Field(Mesh *localmesh) : fieldmesh(localmesh) {
+Field::Field(Mesh *localmesh) : fieldmesh(localmesh), fieldCoordinates(nullptr) {
   if (fieldmesh == nullptr) {
     fieldmesh = mesh;
   }
+
+/// Note we would like to do `fieldCoordinates = getCoordinates();` here but can't
```

Comment:
Isn't that an implementation detail, and thus only have `//` comments?

Reply:
Thanks, you're right.